src/net/ifaddrs/tests.rs

```diff
@@ -0,0 +1,76 @@
+#![cfg(test)]
+
+use std::ptr::null_mut;
+use std::mem::transmute;
+
+use std::ffi::CString;
```

Comment:
Please group all `use std ...` lines together and alphabetize them (so `std::ffi ...` should be first, etc.